### PR TITLE
New verification system just dropped

### PR DESCRIPTION
**Pull request by alexkar598**
TLDR: newverification

Uses my system BAB: Byond Authentication Bridge
Doesn't require any server pr altho it would be a good idea to remove the old dead code for the old system

### CODE REVIEW

File: models/HTTP/RouterPaths/api/RouterPathCallback.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
+const RouterPath = require("../../RouterPath.js");
+const crypto = require("crypto");
+const https = require("https");
+const url = require("url");
+const { URLSearchParams, URL} = require("url");
+const stringUtils = require("../../../Utils/String.js");
+const bodyParser = require("body-parser");
+const escapeHtml = stringUtils.escapeHtml
+
+const errorTpl = `<!DOCTYPE html><html lang="en"><head><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no"><title>Yogstation Account Linking</title><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"></head><body><header class="d-flex justify-content-center align-items-center" style="height: 3rem;background: var(--bs-blue);"><h1 style="color: rgb(255,255,255);">Yogstation Account Linking</h1></header><div class="container" style="margin-top: 2rem;"><div class="card"><div class="card-body d-flex flex-column align-items-center"><h4 class="card-title">An error occured!</h4><p class="card-text">$errormsg$</p></div></div></div><script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"></script></body></html>`
```
Comment (AshCorr) [inline models/HTTP/RouterPaths/api/RouterPathCallback.js:10]: Nice.

**Comment by AshCorr** [comment]
Is Lummox aware that we're intending to start using this in production?

**Comment by alexkar598** [comment]
Yes

File: models/HTTP/RouterPaths/api/RouterPathCallback.js (hunk 1 above)
Comment (alexkar598) [reply models/HTTP/RouterPaths/api/RouterPathCallback.js:10]: I ain't paid enough to figure out how to integrate a template engine in yogbot

**Review by AshCorr — APPROVED** [review]

**Comment by alexkar598** [comment]
*happy alex noises*

**Pull request merged**